great_expectations/datasource/fluent/pandas_datasource.pyi

```diff
@@ -639,7 +639,7 @@ class PandasDatasource(_PandasDatasource):
     ) -> Validator: ...
     def read_html(
         self,
-        io: str | os.PathLike,
+        io: os.PathLike | str,
```

Comment:
Order shouldn't matter here.

Reply:
I wanted everything to match, but in theory this could use `@validate_arguments` although I don't think we would want to.